src/projetos/CadastroProjetosPage.tsx

```
@@ -133,6 +144,19 @@ export function CadastroProjetoPage() {
       if (id) {
         const projeto = await ProjetoService.findById(Number(id))
         if (projeto) {
+          const normalizeStatus = (
+            status: string,
+          ): 'EM_ANDAMENTO' | 'FINALIZADO' | 'CANCELADO' => {
+            const validStatuses = [
+              'EM_ANDAMENTO',
+              'FINALIZADO',
+              'CANCELADO',
+            ] as const
+            return validStatuses.includes(status as any)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)